packages/outline-view/README.md

```diff
@@ -0,0 +1,6 @@
+# Theia - Navigator Extension
```

Comment:
Oh, another navigator extension! :)

Please write two sentences below, what the outline extension does.